solver/src/main.rs

```diff
@@ -25,9 +26,21 @@ struct Arguments {
     #[structopt(short = "k", long, env = "PRIVATE_KEY", hide_env_values = true)]
     private_key: PrivateKey,
 
-    /// The factor by which the gas price estimate is multiplied (to ensure fast settlement)
-    #[structopt(long, env = "GAS_PRICE_FACTOR", default_value = "1.0")]
-    gas_price_factor: f64,
+    /// Which gas estimators to use. Multiple estimators are used in sequence if a previous one
+    /// fails. Individual estimators support different networks.
+    /// `EthGasStation`: supports mainnet.
+    /// `GasNow`: supports mainnet.
+    /// `GnosisSafe`: supports mainnet and rinkeby.
+    /// `Web3`: supports every network.
+    #[structopt(
+        long,
+        env = "GAS_ESTIMATORS",
+        default_value = "Web3",
+        possible_values = &GasEstimatorType::variants(),
+        case_insensitive = true,
+        use_delimiter = true
+    )]
+    gas_estimators: Vec<GasEstimatorType>,
```

Comment:
Is there any way to have mixed gas price averaging or is this a silly idea?

Reply:
On a technical level it would be easy to average sources together but I don't think we need a complex set up like that. The reason for having several sources is that some might be down so it can use the next one in the list.